src/lib/snack-bar/snack-bar-config.ts

```diff
@@ -35,7 +35,13 @@ export class MatSnackBarConfig {
   duration?: number = 0;
 
   /** Extra CSS classes to be added to the snack bar container. */
-  extraClasses?: string[];
+  panelClass?: string[];
+
+  /**
+   * Extra CSS classes to be added to the snack bar container.
+   * @deprecated Use `panelClass` instead.
+   */
+ extraClasses?: string[] = this.panelClass;
```

Comment:
This leaves the option here, but it will still break because it doesn't do anything now; the snack-bar needs to combine both until we remove this one.

Reply:
Done. Checkout what I did there, open to suggestions for better ideas.